### PR TITLE
Automate Publish to GitHub Package

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: scripts/publish-to-github.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo "@dimensiondev:registry=https://npm.pkg.github.com/DimensionDev" > "$HOME/.npmrc"
+
+VERSION=$(jq -r '.version' package.json)
```
Comment (guanbinrui) [inline scripts/publish-to-github.sh:4]: `jq` wasn't listed in `package.json`?

File: package.json [hunk 2]
```diff
@@ -5,29 +5,27 @@
...
-    "bin",
-    "es",
-    "umd"
-  ],
   "scripts": {
     "prebuild": "npm run clean",
-    "build": "npm run build:umd && npm run build:es",
-    "build:es": "tsc -d",
+    "build": "npm run build:umd && npm run build:dist",
+    "build:dist": "tsc && tsc -p tsconfig.esm.json",
```
Comment (guanbinrui) [inline package.json:15]:
```suggestion
    "build": "npm run build:umd && npm run build:esm",
    "build:esm": "tsc && tsc -p tsconfig.esm.json",
```

Comment (septs) [reply package.json:15]: The `build:dist` include `commonjs` and `esmodule` output

File: scripts/publish-to-github.sh (hunk 1 above)
Comment (septs) [reply scripts/publish-to-github.sh:4]: Get the current version.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**